docs/source/testing.rst

```diff
@@ -67,6 +67,17 @@ An (almost) exhaustive example of test mode:
   >>> ts.update_time(time=time, uuid="some-uuid")
   [{'activities': ['hello', 'world'], 'date_worked': '2015-08-07', 'updated_at': '2015-10-18', 'user': 'red-leader', 'duration': 19, 'deleted_at': None, 'uuid': 'some-uuid', 'notes': None, 'project': ['ganeti'], 'issue_uri': 'https://github.com/osuosl/ganeti_webmgr/issues/56', 'created_at': '2014-06-12', 'revision': 2}]
   >>>
+  >>> ts.create_time(time=time)
+  [{'activities': ['docs'], 'deleted_at': None, 'date_worked': '2014-04-17', 'uuid': '838853e3-3635-4076-a26f-7efr4e60981f', 'notes': 'Worked on documentation toward settings configuration.', 'updated_at': None, 'project': 'ganeti_web_manager', 'user': 'example-2', 'duration': '3m', 'issue_uri': 'https://github.com/osuosl/ganeti_webmgr/issues', 'created_at': '2015-05-23', 'revision': 1}]
+  >>>
+  >>> time = {
+  ...    "duration": 300,
```

Comment:
@athai the ``time`` object is for both creating and updating a ``time`` object in TimeSync.

Here, ``time`` needs to be reset with proper fields for each call of ``{update,create}_time``.